### ISSUE TITLE
Error when modifying IP filter in upcloud_managed_database_opensearch resource

### ISSUE THREAD

**Issue by TARHiS** [UpCloudLtd/terraform-provider-upcloud#541]
Hi I got following error when modifying IP addresses in `properties.ip_filter` list in `upcloud_managed_database_opensearch` resource block.
```
error: message="Validation error.", type="https://developers.upcloud.com/1.3/errors#ERROR_INVALID_REQUEST", correlation_id=01HXREDW5WGY79FVXTE0JR83SQ, invalid_params_properties.saml.subject_key='Length must be >= 1, but got 0.', invalid_params_properties.saml.roles_key='Length must be >= 1, but got 0.', invalid_params_properties.openid.refresh_rate_limi
t_time_window_ms='Must be >= 10000 but found 0.', invalid_params_properties.openid.refresh_rate_limit_count='Must be >= 10 but found 0.', invalid_params_properties.openid.jwt_url_parameter='Length must be >= 1, but got 0.', invalid_params_properties.openid.header='Length must be >= 1, but got 0.', invalid_params_properties.openid.jwt_header='Length must be >= 1,
 but got 0.', invalid_params_properties.openid.roles_key='Length must be >= 1, but got 0.' 
 ```

From UpCloud's WebUI I have previously configured OpenSearch SAML and OpenID properties but disabled them afterward.

Here is terraform file I have been using
```tf
resource "upcloud_managed_database_opensearch" "opensearch_example_fi" {
  maintenance_window_dow  = "sunday"
  maintenance_window_time = "05:00:00"
  name                    = "opensearch-example-fi"
  plan                    = "3x2xCPU-14GB-420GB-3D"
  powered                 = true
  title                   = "opensearch-example-fi"
  zone                    = "de-fra1"

  properties {
    automatic_utility_network_ip_filter = true
    ip_filter                           = yamldecode(file("${path.module}/opensearch_allow_ip_list.yml"))
    public_access                       = true
    ism_enabled                         = false
    ism_history_enabled                 = false
    version                             = "2"
  }
}
```

**Comment by kangasta** [comment]
Hello, thank you for the bug report! Seems like a zero vs. empty value bug in the provider. We'll take a look on how to fix that 🤔

**Comment by kangasta** [comment]
This should now be fixed in [v5.5.0](https://github.com/UpCloudLtd/terraform-provider-upcloud/releases/tag/v5.5.0). Please reopen this, if the issue persist.
